invenio_communities/communities/services/service.py

```diff
@@ -72,6 +72,14 @@ def rename(self, id_, identity, data, revision_id=None):
         if 'id' not in data:
             raise ValidationError('Missing data for required field.', 'id')
 
+        data, errors = self.schema.load(
+            data,
+            context={"identity": identity},
+            raise_errors=raise_errors,     # if False, flow is continued with
+            schema_args={'partial': True}  # data only containing valid data,
+                                           # but errors are reported
+        )                                  # (as warnings)
+
```

Comment:
Isn't this problematic, since the `rename` data/payload only has the `id` field (and not e.g. the `metadata`, and `access` fields, which are required)?

Reply:
That's why I think the ` schema_args={'partial': True}` is passed so it doesn't require the whole schema...if I am not mistaken...